commands/connect/state.js

```diff
@@ -5,25 +5,17 @@ const cli = require('heroku-cli-util')
 const co = require('co')
 
 function * run (context, heroku) {
-  context.region = yield regions.determineRegion(context, heroku)
   let connections = yield api.withUserConnections(context, context.app, context.flags, heroku)
 
-  if (!context.flags.resource) {
-    if (context.flags.json) {
```

Comment:
I don't think this line _ever_ worked, insofar as `json` wasn't specified as an available flag.

Reply:
We should probably add that flag wherever we can, but that's not something to worry about in this PR. 👍 for removing dead code while it's dead.

Reply:
Never mind, I now see you already did farther down. 👍 💯